src/main/java/seedu/duke/Parser.java

```diff
@@ -0,0 +1,20 @@
+package seedu.duke;
+
+public class Parser {
+    private static final String EXIT_MESSAGE = "bye";
+    private static final String PASSENGER_ENTITY = "passenger";
+    private static final String FLIGHT_ENTITY = "flight";
+
+    public static void parse(String lineInput) {
+        String[] inputWords = lineInput.split("\\s+");
+        String entity = inputWords[0];
```

Comment:
Do try to add a constant for the value "0" to avoid magic numbers i.e. 0 can be named as a constant "OPERATION_INDEX"